internal/service/shortener.go

```diff
@@ -50,9 +101,15 @@ func (s Shortener) BatchGenerateShortURL(ctx context.Context, URLs []models.Batc
 			OriginalURL: URL.OriginalURL,
 		})
 	}
-	err := s.storage.BatchAddURL(ctx, insertURLs)
+	err := s.storage.BatchAddURL(ctx, userID, insertURLs)
 	if err != nil {
 		return nil, err
 	}
 	return urls, nil
 }
+
+func (s Shortener) BatchDeleteShortURL(ctx context.Context, shortURLs []string) {
```

Comment:
```suggestion
func (s Shortener) BatchDeleteShortURL(_ context.Context, shortURLs []string) {
```
раз контекст не используешь, а сигнатуру функции надо соблюдать

Reply:
Исправил